src/go/rpk/pkg/cli/topic/unmount.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package topic
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/redpanda-data/common-go/rpadmin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newUnmountCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var namespace string
+	var timeout int
+
+	cmd := &cobra.Command{
+		Use:   "Unmount [TOPIC]",
+		Short: "Unmount a topic from the Redpanda Cluster",
+		Long:  helpUnmount,
+		Args:  cobra.MinimumNArgs(1),
+		Run: func(cmd *cobra.Command, topics []string) {
+			pf, err := p.LoadVirtualProfile(fs)
+			out.MaybeDie(err, "rpk unable to load config: %v", err)
+			config.CheckExitCloudAdmin(pf)
+			adm, err := adminapi.NewClient(fs, pf)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			resp, err := adm.AddOutboundMigration(cmd.Context(), rpadmin.OutboundMigration{
+				MigrationType: "outbound",
+				Topics: []rpadmin.Topic{
+					{
+						Topic:     topics[0],
+						Namespace: namespace,
+					},
+				},
+				ConsumerGroups: []string{}, // not implemented yet
+			})
+			out.MaybeDie(err, "unable to add outbound migration: %v", err)
+
+			tmt := time.Duration(timeout * int(time.Minute/time.Nanosecond))
+			if errP := checkMigrationActionAndAdvanceState(cmd.Context(), resp.ID, adm, rpadmin.PrepareMigrationAction, "prepared", tmt); err != nil {
+				out.MaybeDie(errP, "unable to prepare migration", errP)
+			}
+			if errP := checkMigrationActionAndAdvanceState(cmd.Context(), resp.ID, adm, rpadmin.ExecuteMigrationAction, "executed", tmt); err != nil {
+				out.MaybeDie(errP, "unable to execute migration", errP)
+			}
+			if errP := checkMigrationActionAndAdvanceState(cmd.Context(), resp.ID, adm, rpadmin.FinishMigrationAction, "finished", tmt); err != nil {
+				out.MaybeDie(errP, "unable to finish migration", errP)
+			}
+		},
+	}
+	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "Namespace for the topic (optional, defaults to 'kafka')")
```

Comment:
If you agree with the comment on the format usage in `rpk topic mount --from <> --to <>`

I would suggest doing:
```
rpk topic unmount [TOPIC]
```
And parse the topic argument with a namespace: `<Namespace/>TOPIC`. I.e:

```
__redpanda/foo -> Namespace: __redpanda, topic: foo
bar                     -> Namespace: kafka, topic bar
```


also, where in the code are we defaulting to kafka?